src/test/unit/app/controller/courts/PostcodeController.ts

```diff
@@ -636,7 +636,8 @@ describe('PostcodeController', () => {
       slug: testSlug,
       searchValue: '',
       updated: false,
-      errors: [{'text': 'The postcode is already present on the destination court: PL11 1YY,PL1 1'}],
+      errors: [{'text': 'The postcode is already present on the destination ' +
+          'court (your changes have not been saved): PL11 1YY,PL1 1'}],
```

Comment:
When we display the invalid/duplicated/non-existent postcodes, is that better to have a space after the comma?